spec/type_extensions/time_spec.cr

```diff
@@ -0,0 +1,29 @@
+require "../../spec_helper"
+
+describe "UUID column type" do
+  describe ".parse" do
+    it "casts a String successfully" do
+      time = Time.new
```

Comment:
I'd do the `to_s("etc.")` here so it is in the setup phase. What do you think about that?